packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.js

```diff
@@ -12,20 +13,48 @@ import { Item } from "../../List/ListItem";
 import CloseIc from "../../icons/Close";
 import { rtlSpacing, right } from "../../utils/rtl";
 import resolveColor from "./helpers/resolveColor";
-import tooltipArrowStyle from "./helpers/tooltipArrowStyle";
-import resolveTooltipArrowPosition from "./helpers/resolveTooltipArrowPosition";
-import resolveTooltipPosition from "./helpers/resolveTooltipPosition";
-import useDimensions from "../hooks/useDimensions";
-import { POSITIONS, SIDE_NUDGE } from "./consts";
+import resolvePlacement from "./helpers/resolvePlacement";
+import { SIDE_NUDGE } from "./consts";
+import useTheme from "../../hooks/useTheme";
 
 import type { Props } from ".";
 
+const StyledArrow = styled(
+  React.forwardRef(({ className }, ref) => <div className={className} ref={ref} />),
+)`
```

Comment:
We should probably start using `$*` props instead of this. (Not now.)

Reply:
yep, agree